streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDbSessionBytesStoreSupplier.java

```diff
@@ -25,6 +25,8 @@
     private final String name;
     private final long retentionPeriod;
 
+    private static final int NUM_SEGMENTS = 3;
```

Comment:
Why the increase from 2 -> 3 ?

Reply:
`RocksDBSessionStoreSupplier.NUM_SEGMENTS` is 3 -- it's not an increase here, is it?